hack/generate-bindata.sh

```diff
@@ -33,7 +33,8 @@ fi
 export PATH="${KUBE_OUTPUT_BINPATH}:${PATH}"
 
 if ! which go-bindata &>/dev/null ; then
-	echo "Cannot find go-bindata."
+	echo 'Can not find go-bindata, install with:'
```

Comment:
hm, we should be using vendored `go-bindata`. `make generated_files` will build/install `go-bindata`, but I guess this script is maybe called from other places?

Reply:
I followed the steps described by `translations/README.md`, and "Cannot find go-bindata" error occurred when executing `./hack/update-translations.sh`. 
The script `update-translations.sh` ends up calling `generate-bindata.sh`. 
I tried to execute `make generated_files` firstly, but this did not install `go-bindata` to `$GOPATH/bin`. 
Should we guide users to install go-bindata more specifically?
Like:
```
echo 'Can not find go-bindata, install with:'
echo 'go build -o $GOPATH/bin/go-bindata ./vendor/github.com/jteeuwen/go-bindata/go-bindata'
```